p/riscv_test.h

```diff
@@ -140,6 +140,7 @@ handle_exception:                                                       \
   1:    ori TESTNUM, TESTNUM, 1337;                                     \
   write_tohost:                                                         \
         sw TESTNUM, tohost, t5;                                         \
+        fence;                                                          \
```

Comment:
Why is this necessary?

Reply:
It’s not. After consulting with @timsnyder-siv, I am going to close this.

Reply:
@pdonahue-ventana , I talked with Andrew and it isn't.  Thanks for taking a look.